tekton/release-cheat-sheet.md

```diff
@@ -70,19 +77,22 @@ the pipelines repo, a terminal window and a text editor.
     pipeline-release
     ```
 
-8. Watch logs of pipeline-release.
+1. Watch logs of pipeline-release.
+
+1. The YAMLs are now released! Anyone installing Tekton Pipelines will now get the new version. Time to create a new GitHub release announcement:
 
-9. The YAMLs are now released! Anyone installing Tekton Pipelines will now get the new version. Time to create a new GitHub release announcement:
+    1. Choose a name for the new release! The usual pattern is "< cat breed > < famous robot >" e.g. "Ragdoll Norby".
+       (Check the previous releases to avoid repetition: https://github.com/tektoncd/pipeline/releases.)
```

Comment:
I have listed all in a gist [here](https://gist.github.com/pritidesai/4fc02c1adb34e85adb1e88a8392f0340#file-tekton-release-titles-csv) 😱 I think it will be very helpful to make it part of the repo/website somewhere, just the table instead of going through more than dozen different release pages.

Reply:
oh wow that would be really handy to refer to XD